packages/plugins/content/image/tsconfig.json

```diff
@@ -1,8 +1,12 @@
 {
   "extends": "../../../tsconfig.settings.json",
   "compilerOptions": {
+    "downlevelIteration": true,
     "rootDir": "src",
-    "outDir": "lib"
+    "outDir": "lib",
```

Comment:
These file changes actually might not be necessary, I may have been throwing things at the wall while trying to run the example on Windows.